.travis.yml

```diff
@@ -73,6 +73,7 @@ after_success:
   else
    ./gradlew coveralls;
   fi
+- python <(curl -s https://raw.githubusercontent.com/TouK/sputnik-ci/master/sputnik-ci.py)
```

Comment:
It's on a travis node.  I think the worst it can do is probably comment on our pull request, exfiltrate our secret keys,  or possible delete the entire repo depending on how travis's keys are set up.  I wouldn't worry about it.